tests/foreman/ui/test_host.py

```diff
@@ -52,29 +55,28 @@ def setUpClass(cls):
         """
         super(HostTestCase, cls).setUpClass()
         # Create a new Organization and Location
-        cls.org = entities.Organization(name=gen_string('alpha')).create()
-        cls.org_name = cls.org.name
+        cls.org_ = entities.Organization(name=gen_string('alpha')).create()
```

Comment:
That will be hard to remember in the future

How I wish we had namespaced the UI helpers under maybe `cls.ui.<helper`>

The naming seems reasonable though.